lib/evaluate.js

```diff
@@ -1,3 +1,7 @@
+import debugFactory from 'debug'
+
+const debug = debugFactory('spark:evaluate')
```

Comment:
```suggestion
import createDebug from 'debug'

const debug = createDebug('spark:evaluate')
```

Not important, but I would prefer this naming

Reply:
I like that 👍🏻